pkgs/applications/networking/nload/default.nix

```diff
@@ -20,9 +20,10 @@ stdenv.mkDerivation rec {
       name = "nload-0.7.4-Eliminate-flicker-on-some-terminals.patch";
       sha256 = "10yppy5l50wzpcvagsqkbyf1rcan6aj30am4rw8hmkgnbidf4zbq";
     })
-  ];
+  ] ++ lib.optional stdenv.isDarwin ./macos.patch;
```

Comment:
If this patch is available somewhere in the homebrew source repository, this should be a fetchurl from raw.githubusercontent.com

Reply:
brew fetches it from https://sourceforge.net/p/nload/bugs/_discuss/thread/c9b68d8e/4a65/attachment/devreader-bsd.cpp.patch but that patch contains only one of the changes

the other change is embedded inline here, which is very ugly: https://raw.githubusercontent.com/Homebrew/homebrew-core/master/Formula/n/nload.rb

Reply:
I first tried to fetch the one patch. but that failed to apply. so I just combined both in a file

Reply:
The another patch is available from MacPorts. Both apply with `extraPrefix = ""`.